tests/lib/pkgdb.sh

```diff
@@ -97,16 +97,20 @@ distro_install_local_package() {
     case "$SPREAD_SYSTEM" in
         ubuntu-14.04-*|debian-*)
             # relying on dpkg as apt(-get) does not support installation from local files in trusty.
-            eatmydata dpkg -i --force-depends --auto-deconfigure --force-depends-version "$@"
-            eatmydata apt-get -f install -y
+            if [ -$# -gt 0 ]; then
```

Comment:
isn't this what you meant?
```suggestion
            if [ $# -gt 0 ]; then
```